source/MRViewer/MRGLTexture.h

```diff
@@ -49,6 +49,8 @@ class GlTexture
 
     struct Settings
     {
+        // the X and Y components are the dimensions of the target texture
+        // the Z component is the number of layers or textures
```

Comment:
Z - 1 for texture2d
Z - number of textures for texture2d array
Z - Z dimensions for texture3d